src/ESM.sol

```diff
@@ -37,18 +37,19 @@ interface DenyLike {
 }
 
 contract ESM {
-    
+
     uint256 constant WAD = 10 ** 18;
 
     GemLike public immutable gem;   // collateral (MKR token)
     address public immutable proxy; // Pause proxy
 
     mapping(address => uint256) public wards; // auth
     mapping(address => uint256) public sum; // per-address balance
-    
+
     uint256 public Sum; // total balance
     uint256 public min; // minimum activation threshold [wad]
     EndLike public end; // cage module
+    uint256 public stopped;
```

Comment:
why not `live`?

Reply:
modifer is already named live, and it would invert the expected value

Reply:
changed to live at 48147ab4089f21a7639fe9690c3560cd7c1e07eb